codecov.yml

```diff
@@ -0,0 +1,15 @@
+coverage:
+  notify:
+    slack:
+      default:
+        url: "secret:bInnBtjHWqcMiw+KB/NyVFU2yk6gHLdMLmgIdh1wT/VK0OetNc+WP2gYxhQwEy6yoJyRBSmpMfKwn4RuVSEe6orVQNMpt3NS9g53B2btW7a1Iv3SCl+GhPVkVHQiwarhWgFQnbR4QYQyBI5wTOOC1QanuUVn0yQ05FMz/VnMRU8="
```

Comment:
Same here, is this the encrypted secret token?

Reply:
Yes, this one is encrypted using codecov. Not sure if it works, though, as I am yet to receive any notifications. Could be the "threshold: 1%" that prevents notifications.